### PR TITLE
Automate setting up bakerydemo to use the Postgres db in services

### PR DESCRIPTION
**Pull request by cnk**
Added instructions on getting bakerydemo to use the PostgreSQL database from the adjacent container. Also includes how to recognize (and fix) problems due to model changes in wagtail's master branch.

This PR depends on a PR for bakerydemo https://github.com/wagtail/bakerydemo/pull/274

### CODE REVIEW

**Review by zerolab — APPROVED** [review]
I have a minor suggestion, otherwise LGTM

File: README.md [hunk 1]
```diff
@@ -40,18 +43,45 @@ Here is the resulting folder structure:
...
+web_1       |     return self.cursor.execute(sql, params)
+web_1       | psycopg2.errors.UndefinedColumn: column "clean_name" of relation "base_formfield" does not exist
+web_1       | LINE 1: UPDATE "base_formfield" SET "sort_order" = 0, "clean_name" =...
+```
+
+To fix this, open a new terminal window, exec into the container and run makemigrations:
+
+```
+docker exec -it wagtail-dev_web_1 /bin/bash`
+`./manage.py makemigrations`
```
Comment (Scotchester) [inline README.md]: These backticks mess things up now that you have both lines in a fenced code block. Also, a suggestion to clarify where each command is run:

```suggestion
# at the root of your wagtail-dev project folder:
docker exec -it wagtail-dev_web_1 /bin/bash

# then, at the prompt inside the container:
./manage.py makemigrations
```

**Review by Scotchester — COMMENTED** [review]
One small correction and a suggestion.

File: README.md (hunk 1 above)
Comment (cnk) [reply README.md]: Good catch. I remove the stray backticks and explained why you need to rerun `docker-compose up`

**Comment by cnk** [comment]
@zerolab you said you had a suggestion but I don't see what the suggestion is.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,10 @@ git clone git@github.com:wagtail/docker-wagtail-develop.git wagtail-dev
 cd wagtail-dev/
 # 4. Run the setup script. This will check out all the dependency repos.
 ./setup.sh
-# 5. Build the containers
+# 5. Configure your app to use PostgreSQL via the DATABASE_URL from docker-compose.yml
+cp bakerydemo/bakerydemo/settings/local.py.docker-compose-example bakerydemo/bakerydemo/settings/local.py
```
Comment (zerolab) [inline README.md]: Gah, tech fail. The suggestion was to move these into `setup.sh` so the setup part has fewer steps one needs to copy/paste

**Comment by saevarom** [comment]
@cnk could we move that line into setup.sh as per suggestion from @zerolab ?

**Comment by cnk** [comment]
I automated the .env file setup by putting it in the setup.sh file.

Please squash when merging; no need to preserve the previous attempts.

File: README.md (hunk 2 above)
Comment (cnk) [reply README.md]: @zerolab moved the database configuration into the setup.sh script that checks out the repositories

**Comment by saevarom** [comment]
Thanks @cnk! just tested this on a clean computer and everything went very smooth.

**Pull request merged**